.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,40 @@
+name: Question
+description: Submit a question
+labels: ["question"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Asking a question
+        If you have a question about PcapPlusPlus, please check the following resources before opening a new issue:
+        - [PcapPlusPlus documentation](https://pcapplusplus.github.io/docs/quickstart)
+        - [PcapPlusPlus API reference](https://pcapplusplus.github.io/docs/api)
+        - [GIthub issues tracker](https://github.com/seladb/PcapPlusPlus/issues)
```

Comment:
```suggestion
        - [Github issues tracker](https://github.com/seladb/PcapPlusPlus/issues)
```

Reply:
Thanks, let me correct to "GitHub", which is the correct one.